lib/motors/controller.cpp

```diff
@@ -129,4 +133,40 @@ ControllerStatus Controller::processWarningMessage(const std::uint8_t warning_co
   }
   return priority_error;
 }
+
+ControllerState Controller::processNMTMessage(const std::uint8_t nmt_code)
```

Comment:
As someone without context, I'm not clear what nmt is? Is this a common abbreviation or should it either be renamed or have explanatory comments?